### ISSUE TITLE
9.0 compatibility? 

### ISSUE THREAD

**Issue by Avitale34** [crash1115/blind-roll-skills#35]
Is there any plans to update this module to version 9? I really love using this module and it is having some errors with Dice so Nice in V9.

**Comment by crash1115** [comment]
It's in the works, yeah. The module is undergoing a pretty significant overhaul to how it functions behind the scenes, and it is my intent for that update to be working with v9. I don't have an ETA on that at this time, but I'm aiming for next couple weeks.

Thanks for being pleasant about this; patience is always appreciated and I'm glad you're enjoying the module :)

**Comment by Avitale34** [comment]
Hey Crash hate to bother you just wondering if this was still being worked on?

**Comment by crash1115** [comment]
Yup, it's still in progress. Been slammed with other things, not really had the time to work on it that'd I'd have liked.

I'm _pretty_ sure it should work in its current state in v9, unless there's some compatibility issues with other roll modules that I'm unaware of. (If something specific is breaking and preventing you from using it, lemme know.)
